src/cmake/thirdparty/SetupAxomThirdParty.cmake

```diff
@@ -11,27 +11,59 @@
 # UMPIRE
 #------------------------------------------------------------------------------
 if (UMPIRE_DIR)
-    include(cmake/thirdparty/FindUmpire.cmake)
-    blt_register_library( NAME      umpire
-                          INCLUDES  ${UMPIRE_INCLUDE_DIRS}
-                          LIBRARIES umpire
-                          TREAT_INCLUDES_AS_SYSTEM ON)
+    if (NOT EXISTS "${UMPIRE_DIR}")
+        message(FATAL_ERROR "Given UMPIRE_DIR does not exist: ${UMPIRE_DIR}")
+    endif()
+
+    if (NOT IS_DIRECTORY "${UMPIRE_DIR}")
+        message(FATAL_ERROR "Given UMPIRE_DIR is not a directory: ${UMPIRE_DIR}")
+    endif()
+
+    find_package(umpire REQUIRED PATHS ${UMPIRE_DIR} )
+
+    message(STATUS "Checking for expected Umpire target 'umpire'")
+    if (NOT TARGET umpire)
+        message(FATAL_ERROR "Umpire failed to load: ${UMPIRE_DIR}")
+    else()
+        message(STATUS "Umpire loaded: ${UMPIRE_DIR}")
+        set_property(TARGET umpire 
+                     APPEND PROPERTY INTERFACE_SYSTEM_INCLUDE_DIRECTORIES
+                    ${UMPIRE_INCLUDE_DIRS})
+        set(UMPIRE_FOUND TRUE CACHE BOOL "")
+    endif()
 else()
     message(STATUS "Umpire support is OFF")
+    set(UMPIRE_FOUND FALSE CACHE BOOL "")
 endif()
 
 
 #------------------------------------------------------------------------------
 # RAJA
 #------------------------------------------------------------------------------
 if (RAJA_DIR)
```

Comment:
@white238 -- I'd guess the issues with our CI's clang are related to this change.

Reply:
I think it was related to RAJA_DIR pointing at subfolders of the actual install.  Honestly not sure why it worked to begin with but its passing now that I removed them.